build avahi-gobject only when avahi-client is also built

(modified to only patch the makefiles)

Index: trunk/avahi-gobject/Makefile.am
===================================================================
--- trunk/avahi-gobject/Makefile.am	(revision 1669)
+++ trunk/avahi-gobject/Makefile.am	(revision 1670)
@@ -23,6 +23,7 @@
 AM_CFLAGS+='-DDEBUG_TRAP=__asm__("int $$3")'
 
 if HAVE_GOBJECT
+if HAVE_DBUS
 
 avahigobjectincludedir=$(includedir)/avahi-gobject
 
@@ -109,6 +110,7 @@
 	$< > $@
 
 endif
+endif
 
 indent:
 	indent -brf -nbbo -nbc -ip0 -cs -nbfde -npsl -br -brs -bap -i4 -bs -cdw -ce -npcs -hnl -cli4 -nut -ci8 ga-*.[ch]
Index: trunk/Makefile.am
===================================================================
--- trunk/Makefile.am	(revision 1669)
+++ trunk/Makefile.am	(revision 1670)
@@ -112,6 +112,7 @@
 	$(srcdir)/avahi-glib/glib-malloc.h
 
 if HAVE_GOBJECT
+if HAVE_DBUS
 DX_INPUT += \
 	$(srcdir)/avahi-gobject/ga-client.h \
 	$(srcdir)/avahi-gobject/ga-entry-group.h \
@@ -122,6 +123,7 @@
 	$(srcdir)/avahi-gobject/ga-service-resolver.h
 endif
 endif
+endif
 
 if HAVE_DBUS
 DX_INPUT += \
